api/src/main/java/org/openmrs/module/bedmanagement/service/BedManagementService.java

```diff
@@ -191,4 +191,30 @@
      */
     @Authorized(value = "Get Beds", requireAll = true)
     BedLocationMapping getBedLocationMappingByLocationUuidAndRowColumn(String locationUuid, Integer row, Integer column);
+
+    /**
+     * Get bedType by Id
+     *
+     * @param id {@link Integer} Bed type id
+     * @return {@link BedType}
+     */
+    @Authorized(value = {"Get Bed Type"}, requireAll = true)
+    BedType getBedTypeById(Integer id);
```

Comment:
Why is this method necessary? Exposing database IDs should be avoided as much as possible.

Reply:
Because of Bed type entity does not have uuid property.

https://github.com/openmrs/openmrs-module-bedmanagement/blob/ba057324c1a6e742e154e6120cf0c52f078bfd11/api/src/main/resources/BedType.hbm.xml#L5-L17